src/main/java/org/jabref/preferences/WorkspacePreferences.java

```diff
@@ -155,4 +162,12 @@ public BooleanProperty confirmDeleteProperty() {
     public void setConfirmDelete(boolean confirmDelete) {
         this.confirmDelete.set(confirmDelete);
     }
+
+    public ObservableList<String> getSelectedSlrFetchers() {
```

Comment:
```suggestion
    public List<String> getSelectedSlrFetchers() {
```

Reply:
I won't be able to use `.addListener` if I use a list. Will have to make manual update functions. Should I revert this?

Reply:
Ah missed that, then change to observable list here

Reply:
Look in other prefs tabs how lists are handled.